instance/models/mixins/openedx_config.py

```diff
@@ -297,7 +297,17 @@ def _get_configuration_variables(self):
 
             # Insights and analytics_api
             "SANDBOX_ENABLE_ANALYTICS_API": False, # set to true to enable analytics_api
-            "SANDBOX_ENABLE_INSIGHTS": False # set to true to enable insights
+            "SANDBOX_ENABLE_INSIGHTS": False, # set to true to enable insights
+
+            # Disable celery heartbeats
+            # The current default setup of Celery in Open edX uses something called
+            # heartbeats to detect connection drops to the celery broker (RabbitMQ).
+            # With as many 15 celery processes running on Open edX servers this can mean
+            # a lot of the RabbitMQ usage is used up just to check for connection drops,
+            # and that too mostly to get around issues with RabbitMQ behind a load-balancer,
+            # which is not our setup.
```

Comment:
Is this comment talking about a specific instance of Open edX, or all instances that are installed following the default installation guide?

Reply:
By default, all instances provisioned using the EdX configuration repo use the heartbeats.
I've improved the text a little. :)

Reply:
Answering your PR question, you can deploy a RabbitMQ server in some ways depending on your need. 
The heartbeats used by Celery are mainly there to check for connection blocks, and are critical when using RabbitMQ behind a load balancer, which is not our setup. You can read up more about this and how the decision was made by reading the discovery ticket. :grin:

Reply:
Ah, my question was more related to the "behind a load balancer, which is not our setup" part. I'm not sure who "our" refers to here - if not being behind a load balancer is the general case with all OCIM setups, something that's enforced, something that's a default setup but could be modified, or if we should reword it slightly to something like "which is not the standard setup".

Reply:
I'm talking about only the RabbitMQ server: in our case (OpenCraft), we don't use the RabbitMQ server behind a load balancer.

Reply:
I've reworded the comment :grin:

Reply:
Thanks, I think that reads better. I'm conscious that OCIM is open source AGPL software, so potentially is being used by others, which is why I'm a bit cautious about the use of "our" in documentation. :)